packages/dds/tree/docs/SharedTree Philosophy.md

```diff
@@ -252,3 +252,36 @@ Accomplishing this goal requires specialized tree data formats or APIs at a few
     Specialized chunked implementations can declare support for fast path APIs via symbols to allow their user to query for and use them.
 
 To enable efficient encoding and decoding between these formats, both are owned by chunked forest (allowing zero copy optimizations for data arrays between the two), and ref-counting is used on chunks so trees can be lazily cloned, and modified in place when only referenced once.
+
+# Runtime Performance
+
+Where performance costs are incurred is important.
+
+## Application of remote edits vs creation of local edits
+
+SharedTree is designed minimize the cost of applying edits from remote clients since since any client which can't keep up with the remote edit rate is unable to collaborate.
```

Comment:
```suggestion
SharedTree is designed minimize the cost of applying edits from remote clients since any client which can't keep up with the remote edit rate is unable to collaborate.
```